docs/source/Support/bskReleaseNotes.rst

```diff
@@ -41,7 +41,13 @@ Version |release|
   automatically increment the patch number. This allows users to reference/require specific versions of Basilisk 
   outside of the release cycle.
 - updated plotting of ``opNav`` example scenarios to work again with latest version of ``matplotlib``
+<<<<<<< HEAD
```

Comment:
These merge conflict control characters got through. Let's remove them and then this PR is good to go.

Reply:
I just fixed this!